telemetry/statsd/src/main/scala/io/buoyant/telemetry/statsd/StatsDStatsReceiver.scala

```diff
@@ -0,0 +1,48 @@
+package io.buoyant.telemetry.statsd
+
+import java.util.concurrent.ConcurrentHashMap
+import com.timgroup.statsd.StatsDClient
+import com.twitter.finagle.stats.{Counter, Stat, StatsReceiverWithCumulativeGauges}
+import scala.collection.JavaConverters._
+
+private[telemetry] object StatsDStatsReceiver {
+  // from https://github.com/researchgate/diamond-linkerd-collector/
+  private[statsd] def mkName(name: Seq[String]): String = {
+    name.mkString("/")
+      .replaceAll("[^A-Za-z0-9/]", "_")
+      .replace("//", "/")
+      .replace("/", ".") // http://graphite.readthedocs.io/en/latest/feeding-carbon.html#step-1-plan-a-naming-hierarchy
```

Comment:
eh? `.replaceAll("[^A-Za-z0-9/]", "_")` replaces slashes with `_` and then we replace slashes again.  Should replaceAll come after the first replaces?